lib/ansible/modules/network/eos/eos_user.py

```diff
@@ -340,6 +341,12 @@ def main():
                            supports_check_mode=True)
 
     warnings = list()
+    if module.params['password'] and not module.params['configured_password']:
```

Comment:
^ Precisely. I know that people are going to use "password" as the arg for the user's password because that's the intuitive thing to do, so I want to try to warn them when they make that mistake.